cmd/configsum/main.go

```diff
@@ -240,6 +252,50 @@ func metricsRepo() (
 	return repoErrCountFunc, repoOpCountFunc, repoOpLatencyFunc
 }
 
+func metricsRequest() (
+	instrument.CountRequestFunc,
+	instrument.ObserveRequestFunc,
+) {
+	if requestLatency == nil {
+		requestLatency = kitprom.NewHistogramFrom(
+			prometheus.HistogramOpts{
+				Namespace: instrumentNamespace,
+				Subsystem: instrumentSubsystem,
+				Name:      "req_latency_seconds",
+				Help:      "Total duration of requests in seconds",
+			}, requestLabels,
+		)
+	}
+
+	requestLatencyFunc := func(host, method, statusCode string, begin time.Time) {
+		requestLatency.With(
+			labelHost, host,
+			labelMethod, method,
+			labelProto, statusCode,
```

Comment:
what should we track under proto? aren't we always going to use https for configsum?

Reply:
It can be `http` or `https` depending on the setup of each system installation, maybe it has no value at all at the moment and we just drop it until need arises.